core/API/Proxy.php

```diff
@@ -24,15 +25,13 @@
  * object, with the parameters in the right order.
  *
  * It will also log the performance of API calls (time spent, parameter values, etc.) if logger available
- *
- * @method static Proxy getInstance()
  */
-class Proxy extends Singleton
+class Proxy
 {
     // array of already registered plugins names
     protected $alreadyRegistered = array();
 
-    private $metadataArray = array();
+    protected $metadataArray = array();
```

Comment:
not 100% sure if this was meant to be in the PR? Seeing there were couple tests for this but not sure if it was meant for different PR maybe?

Reply:
It's used by the new UI test, see `CustomApiProxy` in UITestFixture.php.